cmd/minishift/cmd/start_preflight.go

```diff
@@ -109,6 +109,14 @@ func preflightChecksBeforeStartingHost() {
 			"Checking if user is a member of the Hyper-V Administrators group",
 			false, configCmd.WarnCheckHyperVDriver.Name,
 			driverErrorMessage)
+	case "virtualbox":
+		preflightCheckSucceedsOrFails(
+			configCmd.SkipCheckVBoxInstalled.Name,
+			checkVBoxInstalled,
+			"Checking if VirtualBox is installed",
+			false, configCmd.WarnCheckVBoxInstalled.Name,
+			driverErrorMessage)
```

Comment:
The driver error message is not appropriate in this case.
```
driverErrorMessage := "See the 'Setting Up the Driver Plug-in' topic (https://docs.openshift.org/latest/minishift/getting-started/setting-up-driver-plugin.html) for more information"
```
As it points to the driver plugin documentation which do not have any information on Virtualbox. It should point to https://docs.openshift.org/latest/minishift/getting-started/installing.html#install-prerequisites

Reply:
Yeah, this should be changed, changing the error message. and about [#1822 comment](https://github.com/minishift/minishift/pull/1822#issuecomment-354450507) the `windows/registry` package is used from`"golang.org/x/sys"`, interestigly i don't see the changes to `gopkg.lock` in my local branch. How do i take care of this?